website/other/bezier-rs-demos/src/App.vue

```diff
@@ -288,6 +288,93 @@ export default defineComponent({
 						},
 					},
 				},
+				{
+					name: "Intersect (Line Segment)",
+					callback: (bezier: WasmBezierInstance): string => {
+						const line = [
+							{ x: 150, y: 150 },
+							{ x: 20, y: 20 },
+						];
+						const mappedLine = line.map((p) => [p.x, p.y]);
+						return bezier.intersect_line_segment(mappedLine);
+					},
+				},
+				{
+					name: "Intersect (Quadratic)",
+					callback: (bezier: WasmBezierInstance, options: Record<string, number>): string => {
+						const quadratic = [
+							{ x: 20, y: 80 },
+							{ x: 180, y: 10 },
+							{ x: 90, y: 120 },
+						];
+						const mappedQuadratic = quadratic.map((p) => [p.x, p.y]);
+						return bezier.intersect_quadratic_segment(mappedQuadratic, options.error);
+					},
+					exampleOptions: {
+						[BezierCurveType.Quadratic]: {
+							sliderOptions: [
+								{
+									variable: "error",
+									min: 0.1,
+									max: 2,
+									step: 0.1,
+									default: 0.5,
```

Comment:
Could you make this slider option a default like we did for `tSliderOption` to reduce duplication? Maybe something like `errorSliderOption`?